runtime/main/src/main/java/org/qbicc/runtime/main/InitialHeap.java

```diff
@@ -0,0 +1,5 @@
+package org.qbicc.runtime.main;
+
+class InitialHeap {
```

Comment:
Using this class as a convenient way to get somewhere to put the section with the initial heap.  Could do something else if there's a better way to accomplish this.  It's a massive amount of data, so don't want to put it in a `.ll` file with anything else we might want to look at.

Reply:
*Not for this PR* but I think the solution here would be to make the enclosing type definition optional on program modules so we can create standalone modules that don't relate to a class.